app/policies/plan_policy.rb

```diff
@@ -56,6 +56,10 @@ def visibility?
     @plan.administerable_by?(@user.id)
   end
 
+  def set_test?
+    @plan.administerable_by?(@user.id)
```

Comment:
Currently update is set as @plan.editable_by?(@user.id)  

Which one is the correct behavior?
I  can see a case for only owners/co-owners to be able to set visibility, test status, and other plan details